test/index.js

```diff
@@ -27,7 +37,8 @@ describe("depcheck", function () {
     var absolutePath = path.resolve("test/fake_modules/good_es6");
 
     depcheck(absolutePath, { "withoutDev": true }, function checked(unused) {
-      assert.equal(unused.dependencies.length, 0);
+      assert.equal(unused.dependencies.length, 1);
+      assert.equal(unused.dependencies[0], "name-import-as");
```

Comment:
It does not provide full context about why `name-import-as` is in the list.

I think it could be better to keep the unsupported line in `good_es6/index.js` and add a comment on it. And, comment something here too.